rebar.config

```diff
@@ -21,21 +21,28 @@
     {cover_tests, [
         {ct, "--sname=ct1 --cover"}
     ]},
+    {find_dead_code, [
+        {do, "default as hank hank"}
+    ]},
     {all, [
         format,
         run_tests,
         xref,
-        dialyzer
+        dialyzer,
+        find_dead_code
     ]}
 ]}.
 
 {profiles, [
     {format, [
         {plugins, [erlfmt]}
     ]},
+    {hank, [
+        {plugins, [rebar3_hank]}
+    ]},
     {test, [
         {plugins, [
-            {rebar3_codecov, "0.4.0"}
+            {rebar3_codecov, "0.5.0"}
         ]}
```

Comment:
Do we need so many profiles? Why can't all of them be in the test profile together? 🤔

Reply:
Because rebar does not do lazy loading for plugins sadly :( 
There is a big chance I could just move all of them into `project_plugins` and don't care about extra 10 seconds per task on CI :D  (later probably)